recipe/meta.yaml

```diff
@@ -24,43 +25,50 @@ requirements:
     - setuptools
     - wheel
   run:
-    - python >=3.7
+    - python
     - appdirs
     - entrypoints
     - jinja2
     - pyyaml
-    - dask
+    - dask-core
     - fsspec >=2021.7.0
```

Comment:
msgpack is also a dependency
https://github.com/intake/intake/blob/0.6.6/requirements.txt#L7

Reply:
It's mentioned below as `msgpack-python`